bootstrap.php

```diff
@@ -48,7 +48,7 @@
 
                 $storages[$key] = [
                     'adapter' => 'League\Flysystem\AwsS3v3\AwsS3Adapter',
-                    'args'    => [$client, $settings['bucket'], $settings['prefix']],
+                    'args'    => [$client, $settings['bucket'], $settings['prefix'], $settings['options']],
```

Comment:
what if no options are provided in the config, then it will cause an error or warning.

could you change it to `$settings['options'] ?? []` or whatever the default value is?

Thanks

Reply:
Good catch, PHP isn't my strong suit!